nipoppy/workflows/pipeline.py

```diff
@@ -149,17 +149,25 @@ def __init__(
         pipeline_step: Optional[str] = None,
         participant_id: str = None,
         session_id: str = None,
+        use_list: Optional[StrOrPathLike] = None,
         hpc: Optional[str] = None,
         write_list: Optional[StrOrPathLike] = None,
         fpath_layout: Optional[StrOrPathLike] = None,
         verbose: bool = False,
         dry_run=False,
     ):
+        if hpc and (write_list is not None):
```

Comment:
The syntax is a bit inconsistent.
```suggestion
        if hpc and write_list:
```